### ISSUE TITLE
Component Presentations using templates from the same region are always collapsed into a single region

### ISSUE THREAD

**Issue by willprice76** [sdl/dxa-web-application-dotnet#26]
If I have a page with the following CPS:

Teaser (2-column)
Teaser (2-column)
Article
Teaser (2-column)
Teaser (2-column)

When the page is rendered, the 2 groups of 2-col teasers will be collapsed into a single 2-col region, whereas what I expected, as an editor is for the article to appear between two separate 2-col regions.

**Comment by rpannekoek** [comment]
If you want to have two Regions which use the 2-Column Region View, you should ensure that you define two Regions with distinct names. For that purpose, we introduced a Region Name metadata field next to Region View Name in DXA 1.2.

**Comment by willprice76** [comment]
This sounds like I would need to duplicate all my component templates - so not an ideal solution - or do I misunderstand?

**Comment by rpannekoek** [comment]
No, that's correct; you will need to have CTs for each distinct Region; the CTs don't represent much more than the MVC metadata.  In general, the currently solution for using CT metadata to indicate Region is not ideal, but a necessary evil until we get full Region support in SDL Web itself (which might happen this year :-))

**Comment by willprice76** [comment]
I created a PR with what I think is a better solution. https://github.com/sdl/dxa-web-application-dotnet/pull/27

**Comment by bb-ceccles** [comment]
Second this as an improvement. Would be a big improvement if this functionality was enabled

**Comment by ablyth-building-blocks** [comment]
+1

**Comment by AndyMcInnes** [comment]
+1

**Comment by rsleggett** [comment]
+1

**Comment by Vouzamo** [comment]
+1 (component presentation metadata would be the best solution)

**Comment by rpannekoek** [comment]
TSI-1945 (internal issue ID for tracking purposes)

**Comment by justsayantan** [comment]
+1
